ha/k8s_setup/ha_setup.py

```diff
@@ -337,15 +337,19 @@ def process(self):
         except TypeError as type_err:
             Log.error(f'HA config command failed: Type mismatch: {type_err}.\n')
             sys.stderr.write(f'HA config command failed: Type mismatch: {type_err}.\n')
+            raise
```

Comment:
instead of re-raising, cannot we return an error code?

Reply:
This exception gets caught by the caller:
https://github.com/Seagate/cortx-ha/pull/721/files#diff-3961fe8f57cf6e98896333df1e8846fb536f6a2b6384bc51019b15822a432913R587
And there, the return code is returned. Here it was not raised and that is why there was a problem. So, I fixed it by re-raising.
The return code will definitely be going to return from now.

Reply:
Okay.